analyzers/src/SonarAnalyzer.CSharp/Rules/InfiniteRecursion.cs

```diff
@@ -65,6 +65,14 @@ protected override void Initialize(SonarAnalysisContext context)
                     }
                 },
                 SyntaxKind.PropertyDeclaration);
+
+            context.RegisterSyntaxNodeActionInNonGenerated(
```

Comment:
nitpick: this could be grouped with the other registrations above that use `CheckForNoExitMethod`

(currently there are a few registrations with `CheckForNoExitMethod`, then one registration with `CheckForNoExitProperty`, then `CheckForNoExitMethod` again )